packages/core/admin/admin/src/core/apis/Widgets.ts

```diff
@@ -25,6 +25,7 @@ type WidgetArgs = {
   pluginId?: string;
   id: string;
   permissions?: Array<Pick<Permission, 'action'> & Partial<Omit<Permission, 'action'>>>;
+  superAdminOnly?: boolean;
```

Comment:
Just wondering if it would not be more flexible to have an array of roles instead of a boolean for a single role.